spec/Appendix C -- Schema Coordinates.md

```diff
@@ -0,0 +1,93 @@
+# Appendix: Schema Coordinates
+
+Schema Coordinates are human readable strings that uniquely identify an element defined in a GraphQL Schema.
+
+## Definition 
+
+SchemaCoordinates :
+  - TypeDefinitionName FieldSpecifier?
+  - EnumName EnumValueSpecifier?
+  - @ DirectiveName ArgumentSpecifier?
+  - UnionName
+
+TypeDefinitionName:
+  - ObjectTypeName
+  - InterfaceTypeName
```

Comment:
Don't forget input types, also these names aren't defined.

Since this is the grammar, I think it's actually more like:

```
SchemaCoordinate:
  - Name FieldSpecifier?
  - @ Name ArgumentSpecifier?

FieldSpecifier:
  - . Name ArgumentSpecifier?

ArgumentSpecifier:
  - ( ArgumentName : )
```

or maybe:

```
SchemaCoordinate:
  - TypeCoordinate
  - FieldCoordinate
  - FieldArgumentCoordinate
  - DirectiveCoordinate
  - DirectiveArgumentCoordinate

TypeCoordinate: Name

FieldCoordinate: Name . Name

FieldArgumentCoordinate: Name . Name ArgumentSpecifier

DirectiveCoordinate: @ Name

DirectiveArgumentCoordinate @ Name ArgumentSpecifier

ArgumentSpecifier: ( ArgumentName : )
```

I'm not happy using 'FieldCoordinate' to refer to enum values too, but since they use the same syntax they're seen as the same thing by the grammar. (Perhaps this is a good reason to use `EnumType->ENUM_VALUE` or similar to refer to them?)

Reply:
Great clarification @benjie. I like the descriptiveness of the bottom one, but the top one is definitely more compact.

> I'm not happy using 'FieldCoordinate' 

`FieldOrEnumCoordinate` might make that less awkward?

> `ArgumentSpecifier: ( ArgumentName : )`

Should probably be `ArgumentSpecifier: ( Name : )`

Reply:
Oops yeah!

Reply:
resolving this in favor of this thread: https://github.com/graphql/graphql-spec/pull/794#discussion_r527081282